src/main/Port.Adapter/UI/Views/Blazor.Common/TreeView.razor

```diff
@@ -138,9 +139,9 @@
                                 @if (!this.ShouldRightJustifyBasedOnRoot(child))
                                 {
                                     @if (
-                                        !string.IsNullOrWhiteSpace(child.Neuron.ExternalReferenceUrl) ||
-                                        (child.Neuron.Terminal != null && !string.IsNullOrWhiteSpace(child.Neuron.Terminal.ExternalReferenceUrl))
-                                    )
+                        !string.IsNullOrWhiteSpace(child.Neuron.ExternalReferenceUrl) ||
```

Comment:
Can the tabs be restored so it is aligned with the rest of the code?

Reply:
Done